velox/functions/prestosql/aggregates/tests/SumDataSizeForStatsTest.cpp

```diff
@@ -217,7 +217,7 @@ TEST_F(SumDataSizeForStatsTest, complexRecursiveGlobalAggregate) {
       }),
   })};
 
-  testAggregations(vectors, {}, {"sum_data_size_for_stats(c0)"}, "SELECT 118");
+  testAggregations(vectors, {}, {"sum_data_size_for_stats(c0)"}, "SELECT 115");
```

Comment:
just curious, how are these affected?

Reply:
Found the answer in one of your comments that you added internally. Thanks for the clarification:
> The sum_data_size_for_stats tests needed to be updated because they rely on PrestoVectorSerde's estimateSerializedSize function.  It was incorrectly estimating the size of a null array as 4 bytes (the length) but it is actually 1 byte (the null bit mask), so I had to subtract 3 in a bunch of places.
This fixes sum_data_size_for_stats to be closer to Presto (which estimates 0 bytes for null).  The inconsistencies between Velox and Presto are a known issue https://github.com/prestodb/presto/issues/20909 (note that Presto's implementation is not tied to the serializer).